<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-family: &quot;微软雅黑&quot;;
	background-color:#383934;
	border-radius: 5px;
	color: white
}
#groupBox{
	background-color:#707070
}
#top_info_window{
	border-radius: 300px
}
#title{
	color: rgb(151, 190, 21);
	font: bold 25px;
	margin-top:-5px;
	background-color:#707070

}
#ZOOMLION{
	color: rgb(151, 190, 21);
	font: bold 40px;
	background-color:#707070;
	margin-bottom: -5px;
	padding-bottom: -5px;
}
#console_window{
	background-color:#707070;
}

QPushButton{ 
    color: white ;
    background-color: rgb(151, 190, 21) ;
    font-size:15px;
	height: 35px;
	width: 60px;
}
QPushButton:hover{ 
    color: white ;
    background-color: rgb(151, 190, 21) ;
    font-size:15px;
	height: 35px;
	border:5px solid white
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,111111000">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,4,2,2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="ZOOMLION">
           <property name="layoutDirection">
            <enum>Qt::LayoutDirection::LeftToRight</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Plain</enum>
           </property>
           <property name="text">
            <string>ZOOMLION</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="title">
           <property name="text">
            <string>视频全景图拼接系统</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
           <property name="margin">
            <number>-2</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>194</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTextBrowser" name="top_info_window">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="lineWrapMode">
            <enum>QTextEdit::LineWrapMode::FixedPixelWidth</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>设置(未实装）</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="8,2">
      <item>
       <widget class="QGraphicsView" name="big_graphic_window"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1,4,1,1">
        <item>
         <widget class="QGraphicsView" name="small_graphic_window"/>
        </item>
        <item>
         <widget class="QPushButton" name="switch_button">
          <property name="text">
           <string>切换窗口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="console_window">
          <property name="tabletTracking">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="select_button">
          <property name="text">
           <string>选择视频</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="start_button">
          <property name="text">
           <string>开始拼接</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
